packages/platform/src/editor/plugins/BackslashPlugin.tsx

```diff
@@ -0,0 +1,83 @@
+import { computePosition } from "@floating-ui/dom";
+import { useLexicalComposerContext } from "@lexical/react/LexicalComposerContext";
+import { $getSelection, $isRangeSelection } from "lexical";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { createPortal } from "react-dom";
+import { BackslashMenu, BackslashMenuCoords } from "../components/BackslashMenu/BackslashMenu";
+import { useBackslashMenu } from "../components/BackslashMenu/useBackslashMenu";
+
+const DOM_ELEMENT = document.body;
+
+export default function FloatingMenuPlugin() {
+  const ref = useRef<HTMLDivElement>(null);
+  const [coords, setCoords] = useState<BackslashMenuCoords>(undefined);
+  const [editor] = useLexicalComposerContext();
+  const [backslashOpen, setBackslashOpen] = useState(false);
+
+  const { isKeyDown, isKeyReleased } = useBackslashMenu();
+
+  const calculatePosition = useCallback(() => {
+    const domSelection = getSelection();
+    const domRange = domSelection?.rangeCount !== 0 && domSelection?.getRangeAt(0);
+    if (!domRange || !ref.current) {
+      return setCoords(undefined);
+    }
+    computePosition(domRange, ref.current, { placement: "bottom-start" })
+      .then((pos) => {
+        setCoords({ x: pos.x, y: pos.y - 10 });
```

Comment:
nit: it can be helpful to give this magic number (10) a const name to explain what it is being used for, e.g. `const MENU_HORIZONTAL_INDENT = 10;`. I don't know if this is actaully an indent value but the name would explain that.